pallets/funding/src/lib.rs

```diff
@@ -18,96 +18,51 @@
 //!
 //! Polimec's main business logic. It allows users to create, evaluate, and fund projects.
 //!
-//! It rewards project evaluators and contributors with `Contribution Tokens`. These tokens
-//! can be redeemed for a project's native tokens, after their parachain is deployed on mainnet.
-//! ## 👷 Work in progress 🏗️
-//! Expect major changes between PRs
+//! It rewards project participants with Contribution Tokens. These tokens
```

Comment:
Participants get contribution tokens. Contribution tokens are tokens issued by projects which successfully raised funds on Polimec. They are distributed to evaluators and participants who contributed to the project’s successful funding round. Contribution tokens are transferability-locked in the wallet of participants or evaluators after distribution and are automatically converted to the project’s transferable mainnet token at launch.